README.md

```diff
@@ -521,13 +569,81 @@ You can access the service using the same cURL commands that you used above.
+Refer to [Ballerina_Kubernetes_Extension](https://github.com/ballerinax/kubernetes) for more details and samples on Kubernetes deployment with Ballerina. You can also find details on using Minikube to deploy Ballerina programs.
+
+#### company_recruitment_agency_service.bal
+
+Let's now see how we can deploy our `company_data_service` on Kubernetes.
+ 
+First you need to import `ballerinax/kubernetes` and use `@kubernetes` annotations as shown below to enable Kubernetes deployment for the service you developed above.
+
+```ballerina
+import ballerinax/kubernetes;
+
+@kubernetes:Ingress {
+    hostname: "ballerina.guides.io",
+    name: "ballerina-guides-company_data_service",
+    path: "/"
+}
+
+@kubernetes:Service {
+    serviceType: "NodePort",
+    name: "ballerina-guides-company_data_service"
+}
+
+@kubernetes:Deployment {
+    image: "ballerina.guides.io/company_data_service:v1.0",
+    name: "ballerina-guides-company_data_service"
+}
+endpoint http:Listener listener {
+    port: 9090
+};
+
+// Company data management is done using an in memory map.
+map<json> companyDataMap;
+
+// RESTful service.
+@http:ServiceConfig { basePath: "/companies" }
+service<http:Service> orderMgt bind listener {
+``` 
+- Here we have used the `@kubernetes:Deployment` annotation to specify the name of the Docker image that is created as part of building this service.
+- We have also specified `@kubernetes:Service` so that it will create a Kubernetes service that exposes the Ballerina service running on a Pod.
+- The `@kubernetes:Ingress` annotation is for the external interface to access your service (with path `/` and host name `ballerina.guides.io`).
+
+If you are using Minikube, you need to set a couple of additional attributes to the `@kubernetes:Deployment` annotation.
+- `dockerCertPath` - The path to the certificates directory of Minikube (e.g., `/home/ballerina/.minikube/certs`).
+- `dockerHost` - The host for the running cluster (e.g., `tcp://192.168.99.100:2376`). The IP address of the cluster can be found by running the `minikube ip` command.
+
+Now you can build a Ballerina executable archive (.balx) of the service that you developed above using the following command. This will also create the corresponding Docker image and the Kubernetes artifacts using the Kubernetes annotations that you have configured above.
+  
+```
+   $ ballerina build company_data_service
+   
+   @kubernetes:Service                      - complete 1/1
+   @kubernetes:Ingress                      - complete 1/1
+   @kubernetes:Docker                       - complete 3/3 
+   @kubernetes:Deployment                   - complete 1/1
+  
+   Run following command to deploy kubernetes artifacts:  
+   kubectl apply -f ./target/kubernetes/company_data_service
+```
+You can verify Kubernetes deployment, service and ingress are running properly, by using following Kubernetes commands.
```

Comment:
I think "ingress" should also be title case here. 

Also, let's add serial commas to a list like this. Basically in lists, you need to have a comma before "and" as well.